tests/security/selinux/audit2allow.pm

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test "# audit2allow" command with options
+#          "-a / -i / -w / -R / -M / -r" can work
+# Maintainer: llzhao <llzhao@suse.com>
+# Tags: poo#61792, tc#1741285
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use utils;
+
+sub run {
+    my ($self)      = @_;
+    my $testfile    = "test_file";
+    my $test_module = "test_module";
+    my $audit_log   = "/var/log/audit/audit.log";
+
+    $self->select_serial_terminal;
+
+    assert_script_run("systemctl restart auditd");
+    #assert_script_run("echo > /var/log/audit/audit.log");
```

Comment:
should delete these two line if we do not need execute it?

Reply:
Deleted, thanks :)